docs-source/content/userguide/introduction/introduction.md

```diff
@@ -23,18 +23,49 @@ Detailed instructions are available [here]({{< relref "/userguide/managing-opera
 * Oracle WebLogic Server 12.2.1.3.0 with patch 29135930.
```

Comment:
We should mention that Oracle WebLogic versions 12.2.1.4 and later do not require a patch.

Reply:
have we published a 12.2.1.4 image yet?

Reply:
I don't see one, or any samples in the docker-images project - I would rather we fixed that first

Reply:
12.2.1.4 content can be added in another MR